project_templates/stack_package/hooks/post_gen_project.py

```diff
@@ -58,10 +58,11 @@
         print('(post-gen hook) Removing {0} directory'.format(dirname))
         shutil.rmtree(dirname, ignore_errors=True)
 
-    # Remove C++ -related documentation files
-    doc_files = [os.path.join('doc', n)
-                 for n in ('SConscript', 'doxygen.conf.in')]
-    for filename in doc_files:
+    # Remove C++ -related files
+    cpp_files = (os.path.join('doc', 'SConscript'),
+                 os.path.join('doc', 'doxygen.conf.in'),
+                 os.path.join('ups', '{}.cfg'.format(package_name)))
```

Comment:
Consider an f string here: `f'{package_name}.cfg'`

Reply:
This is running in Python 3.6 containers at the moment, so it's not worth the change.